### DISCUSSION TITLE
Can't seem to add any component using the `blender_bevy_toolkit` Blender extension

### DISCUSSION THREAD

**Discussion by Joe23232** [bevyengine/bevy#2798] [Q&A]
Hey guys so I am using this extension for Blender [`blender_bevy_toolkit`](https://github.com/sdfgeoff/blender_bevy_toolkit) which allows you to export your scene to the Blender Render.  I can't seem to add any component to an object

![image](https://user-images.githubusercontent.com/34926497/132697812-00e25474-d15f-4183-b4d5-4ec8a0a3b01e.png)

Not too sure why this is happening?

**Comment by bjorn3** [comment]
`blender_bevy_toolkit` is an unofficial plugin. You are more likely to get a response at https://github.com/sdfgeoff/blender_bevy_toolkit/issues/10.

**Reply by Joe23232** [reply]
Thanks mate
